content/html/concepts/entities/terms/greek-symbols/greek-symbols.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: "Greek Symbols"
+Description: "Greek symbols are frequently employed in scientific, mathematical, and technological applications. They can be used via HTML entities."
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Characters"
+  - "Symbol"
+CatalogContent:
+  - "learn-html"
+  - "paths/front-end-engineer-career-path"
+---
+
+Greek symbols are frequently employed in scientific, mathematical, and technological applications. They can be used via HTML entities and by using Unicode values.
+
+## Example
+
+Using the HTML entity to display the Greek letter "alpha":
+
+```
+&alpha;
+```
+
+The following format represents the Greek letter "alpha" in Unicode:
+
+```
+&#x03B1;

```

Comment:
which type of picture @yangc95 ? like pictures of the GREEK letters?

Reply:
yeah, exactly. Just one small one will do

Reply:
@Sushilverma002 actually, it seems a bit redundant since the table below shows even more examples.

We don't need to stick so closely to the term template for this one. Let's remove this example section and keep the tables below instead

Reply:
okaayy

Reply:
I think there was a misunderstanding 😆 The original tables were good, didn't mean for all your table work to be scratched

Can you bring those tables back and add a small image in replacement for what you have on line 28?